### DISCUSSION TITLE
Option to create avatar instead of image

### DISCUSSION THREAD

**Discussion by benhaynes** [directus/directus#3123] [Feature Requests]
I kinda love this generator, and it works on query strings so we could build it as an additional option into the upcoming "Avatar" interface.

https://getavataaars.com/?accessoriesType=Blank&avatarStyle=Circle&clotheColor=Gray01&clotheType=ShirtVNeck&eyeType=Default&eyebrowType=RaisedExcitedNatural&facialHairColor=Blonde&facialHairType=BeardLight&graphicType=Hola&hairColor=BrownDark&mouthType=Smile&skinColor=Tanned&topType=ShortHairShortFlat

<img width="255" alt="Screen Shot 2020-08-17 at 9 05 46 PM" src="https://user-images.githubusercontent.com/522079/90506845-e8420580-e122-11ea-82ca-31087fc8486c.png">

Many people don't like uploading images of themselves (waste of time, privacy reasons, no good photo, etc)... but this would help differentiate between users a bit more OOTB. I think this service is a good balance of reasonably professional, but not so boring as the "initials in a circle" (eg: `(BH)`).

For the Avatar interface, I imagine using the standard "drop your image here" image interface... but adding this as an option that opens in a modal or expands on-page.

Thoughts?

**Comment by rijkvanzanten** [comment]
<img width="255" alt="Screen Shot 2020-08-17 at 9 05 46 PM" src="https://user-images.githubusercontent.com/9141017/90465817-6c699e00-e0cd-11ea-9642-4f9a61a331b7.png">

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
